ArduCopter/control_auto.cpp

```diff
@@ -671,10 +674,9 @@ void Copter::set_auto_yaw_look_at_heading(float angle_deg, float turn_rate_dps,
     // get turn speed
     if (is_zero(turn_rate_dps)) {
         // default to regular auto slew rate
-        yaw_look_at_heading_slew = AUTO_YAW_SLEW_RATE;
+        yaw_look_at_heading_slew_cdps = AUTO_YAW_SLEW_RATE * 100;
```

Comment:
I think it should be better to change AUTO_YAW_SLEW_RATE to centi degree to avoid calculation!